README.md

```diff
@@ -142,15 +142,16 @@ You can deploy this project adding new lines of code to index.html and styles.cs
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 Habtamu Alemayehu
+👤 Habtamu Alemayehu 
 - GitHub: [@bena](https://github.com/Benawi)
-
+👤 Jubaer hossain maruf
+- GitHub:[jubaer919] (https://github.com/jubaer919)
```

Comment:
- [OPTIONAL] KIndly address this. 👇🏻 

> ```
> ## 👥 Authors <a name="authors"></a>
> 
> 👤 Habtamu Alemayehu
> - GitHub: [@bena](https://github.com/Benawi)
> 
> 👤 Jubaer hossain maruf
> - GitHub:[jubaer919] (https://github.com/jubaer919)
> ```
> 
> **[OPTIONAL]**
> 
> * [ ]  It would be nice if you make your Authors section as you can see below to make your readme look more professional +1
> * [ ]  It would be great if you add more accounts such as Twitter and LinkedIn in this section to give your viewers access to talk with you on multiple platforms blush